go.mod

```diff
@@ -3,8 +3,14 @@ module github.com/kedacore/http-add-on
 go 1.16
 
 require (
+	github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6 // indirect
```

Comment:
@ajanth97 I'm not seeing why all these new dependencies need to be added. could you try running `go mod tidy`? it _should_ get rid of these

Reply:
Yes, that fixed it